### PR TITLE
import UserDataPayload from dp-jwt-verifier-java

### PR DESCRIPTION
**Pull request by Marianasilvago**
### What
Import UserDataPayload from JWT verifier

### How to review
review code changes

### Who can review
Anyone

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+.idea
```
Comment (cookel2) [inline .gitignore]: A new line is needed at the end of this file.

**Review by cookel2 — APPROVED** [review]

**Review by cookel2 — CHANGES_REQUESTED** [review]
see comment

File: src/test/java/com/github/onsdigital/dp/authorisation/permissions/CheckerTest.java [hunk 2]
```diff
@@ -63,7 +63,7 @@ public void setUp() throws Exception {
     @Test
     public void testHasPermission() throws Exception {
         Boolean hasPermission = checker.hasPermission(
-                new UserDataPayload("userId", new String[]{"admin"}), "users.add", null
+                new UserDataPayload("userId", "userEmail", Arrays.asList(new String[]{"admin"})), "users.add", null
```
Comment (rafahop) [inline src/test/java/com/github/onsdigital/dp/authorisation/permissions/CheckerTest.java]: We don't really need to create an array when using the `Arrays.asList` and could use var args:
```suggestion
                new UserDataPayload("userId", "userEmail", Arrays.asList("admin")), "users.add", null
```

File: .gitignore (hunk 1 above)
Comment (Marianasilvago) [reply .gitignore]: New line added

File: src/test/java/com/github/onsdigital/dp/authorisation/permissions/CheckerTest.java (hunk 2 above)
Comment (Marianasilvago) [reply src/test/java/com/github/onsdigital/dp/authorisation/permissions/CheckerTest.java]: Fixed redundant array creation for calling varargs method.
Please review it again, thanks for the suggestion.

**Review by rafahop — APPROVED** [review]

**Review by cookel2 — APPROVED** [review]

**Pull request merged**